climada/util/api_client.py

```diff
@@ -862,9 +862,13 @@ def get_litpop(self, country=None, exponents=(1,1), version=None, dump_dir=SYSTE
         elif isinstance(country, str):
             properties['country_name'] = pycountry.countries.lookup(country).name
         elif isinstance(country, list):
+            if len(set(country)) > 1:
+                raise ValueError("``get_litpop`` can only query single countries. Download the"
+                                 " data for multiple countries individually, and concatenate the"
+                                 " objects using ``LitPop.concat``")
```

Comment:
Can you add this information to the docstring as well, maybe with some example code?